### ISSUE TITLE
Version 2.21.2 (spring-23) contains a blob as readme and breaks all builds

### ISSUE THREAD

**Issue by samuelstein** [salesforce-ux/design-system#756]
We discovered a problem with the **@salesforce-ux/design-system@2.21.2**. The readme contains a binary blob instead of text:

![image](https://github.com/salesforce-ux/design-system/assets/3177158/4659205e-783d-432b-846f-5efcff3659c5)

Thats why our package registry nexus can not marshal the binary into a string and therefore builds with this version are failing.

Could you please upload the library again?

**Comment by transacid** [comment]
this is an issue for e.g. nexus be cause of:
```
$ curl -s https://registry.npmjs.org/%40salesforce-ux/design-system | jq '.versions."2.21.2".readme' -r | wc -c
 17811996
```
nexus cannot marshal the readme field:
```
2023-08-28 09:28:34,739+0000 WARN  [qtp1548545460-5912] admin org.sonatype.nexus.repository.httpbridge.internal.ViewServlet - Failure servicing: GET /repository/npm-group/%40salesforce-ux/design-system
com.fasterxml.jackson.core.exc.StreamConstraintsException: String length (5046272) exceeds the maximum length (5000000)
```

**Comment by transacid** [comment]
for reference https://www.npmjs.com/package/@salesforce-ux/design-system/v/2.21.2

**Comment by samuelstein** [comment]
Can you plese have a look @brandonferrua?

**Comment by Dottenpixel** [comment]
Thanks for your post @samuelstein! We caught that shortly after the release and released `2.21.3`. Please update your dependency to point to at least `2.21.3`.
